### DISCUSSION TITLE
Bar/Line Chart Display

### DISCUSSION THREAD

**Discussion by jr200** [visgl/deck.gl#8358] [Q&A]
Hi,

Under the main website: showcase -> Radiance, and also in Marti Percay's talk at FOSS4G on ([youtube](https://www.youtube.com/watch?v=EuymvVuV0K4&t=7s)), I see a charting overlay is being used while data is being selected.

What component is this - is there an example somewhere? Alternatively, could someone give me some suggestions on the direction to implement this myself. I'm aware of the widgets feature being implemented for v9, but I think I might need something up and running a bit sooner, even if it is a bit rough and ready...

Thanks,

**Comment by majid-saeed** [comment]
Have you got anything in this regard?

**Reply by jr200** [reply]
I ended up implementing this manually using recharts - which was responsive enough for my small dataset (~1000 data points after aggregations).
